docs/introduction/overview.md

```diff
@@ -15,6 +15,7 @@ Phoenix is actually the top layer of a multi-layer system designed to be modular
 Phoenix is made up of a number of distinct parts, each with its own purpose and role to play in building a web application. We will cover them all in depth throughout these guides, but here's a quick breakdown.
 
  - [Endpoint](endpoint.html)
+    - where external requests are received and internal processing begins
```

Comment:
I think I would prefer something discussing that this is the application entry point. All connections start here. "Thinking of it as the original pipeline"